web_responsive/static/src/less/form_view.less

```diff
@@ -1,14 +1,47 @@
 /* Copyright 2016 Ponto Suprimentos Ltda.
+   Copyright 2018 Alexandre Díaz
  * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
 
+@sheet-margin: @sheet-padding;
+@chatter-side-width: @screen-md / 2.5;
+
+// Sided Chatter
+@media (min-width: @screen-md) {
+    .o_chatter_position_sided {
+        .o_form_view {
+            display: flex;
+            height: 100%;
+
+            .o_form_sheet_bg {
+                border-right: 1px solid @table-border-color;
+                overflow: auto;
+                flex: 1 1 auto;
```

Comment:
Use different grow & shrink values for me makes the view "inconsistent" ... 
![2018-06-20-153006_1366x768_scrot](https://user-images.githubusercontent.com/731270/41661558-4ee08304-749f-11e8-970c-6f664ef50eda.png)
![2018-06-20-153003_1366x768_scrot](https://user-images.githubusercontent.com/731270/41661559-4f0f599a-749f-11e8-8401-df52ea611bee.png)
![2018-06-20-153000_1366x768_scrot](https://user-images.githubusercontent.com/731270/41661560-4f35348a-749f-11e8-9945-4ec6fa5bea55.png)
![2018-06-20-152927_1366x768_scrot](https://user-images.githubusercontent.com/731270/41661562-4f50fa6c-749f-11e8-81c9-caff7ea93940.png)

Reply:
Well, the point is that i.e. here I'm working with an ultrawide monitor. Under this circumstance, when window is full width, the chatter is extremely small. Emails sometimes have fixed widths, and with such a big monitor, it seems more comfortable to make it grow a little bit when the screen is big enough. Instead of making it feel unconsistent, here it feels more responsive.

However, if you prefer, you can combine `flex` with `min-width` and `max-width`. Play with those as you like. For instance, setting the chatter as 1/3 of the window, with `min-width: @chatter-side-width`. Do you prefer if I add the commit myself, since I have this monitor to test with?

Reply:
Ok, i changed width to use percentage instead fixed width...
Feel free to commit changes what you think appropriate.